packages/p2p/src/components/order-details/order-details-confirm-modal.jsx

```diff
@@ -27,20 +27,42 @@ const OrderDetailsConfirmModal = ({
     const isMounted = useIsMounted();
     const [error_message, setErrorMessage] = React.useState('');
     const [is_checkbox_checked, setIsCheckboxChecked] = React.useState(false);
+    const [is_process_request, setIsProcessRequest] = React.useState(false); // This state disables the Release amount button during a request
 
     const confirmOrderRequest = () => {
+        setIsProcessRequest(true);
         requestWS({
             p2p_order_confirm: 1,
             id,
-        }).then(response => {
-            if (isMounted()) {
-                if (response.error) {
-                    setErrorMessage(response.error.message);
+        })
+            .then(response => {
```

Comment:
Is the code formatted? 
Please use prettier to format the code.

Reply:
I am using prettier. For some reason, thats how the code is refactored

Reply:
@likhith-deriv have you run `npm run prettify`?😊

Reply:
Yes I have. But in this file, line 38, still seems to be kept in that way. If I remove the spaces and move it up, on save it reverts back :(